pom.xml

```diff
@@ -425,6 +425,18 @@
                                                 <CAMEL_K_CONF_D>target/test-classes/camel-k-runtime/conf.d</CAMEL_K_CONF_D>
                                             </environmentVariables>
                                         </test>
+                                        <test>
+                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-aws2-grouped:${camel-quarkus-test-list.version}</artifact>
+                                            <systemProperties>
+                                                <localstack.container.image>localstack/localstack:3.0.1</localstack.container.image>
+                                            </systemProperties>
+                                        </test>
+                                        <test>
+                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-aws2-quarkus-client-grouped:${camel-quarkus-test-list.version}</artifact>
+                                            <systemProperties>
+                                                <localstack.container.image>localstack/localstack:3.0.1</localstack.container.image>
```

Comment:
@ppalaga FYI, I overrode the localstack version to address the change in the SQS protocol with recent AWS SDK versions. This override can be removed once the localstack version is updated on the Quarkus Camel side.

Reply:
> This override can be removed once the localstack version is updated on the Quarkus Camel side

I took care of it here for 3.7.0 https://github.com/apache/camel-quarkus/pull/5558.